lib/identity/auth.rb

```diff
@@ -238,8 +235,12 @@ def call_authorize
       @cookie.authorize_params = nil
 
       begin
-        # have the user login if we have no session for them
-        raise Identity::Errors::NoSession if !@cookie.access_token
+        # Have the user login if:
+        # - We have no session for them
+        # - The client requested that they login
+        if !@cookie.access_token || params[:prompt] == 'login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.